packages/odata/src/interfaces/odataOption.interface.ts

```diff
@@ -7,6 +7,21 @@ export interface OdataOption extends BackendServiceOption {
   /** Add the total count $inlinecount (OData v2) or $count (OData v4) to the OData query */
   enableCount?: boolean;
 
+  /**
+   * Query fields using $select. The row identifier field is always added.
+   * E.g.: columns [{ field: 'date' }] results in $select=id,date
+   */
+  enableSelect?: boolean;
+
+  /** Which field to use as the row identifier when using enableSelect */
+  rowId?: string;
```

Comment:
I'm not sure about this name here, I find it's too much related to a row and maybe not enough to the data. Can it be `idProperty`, `idPropName`, `dataId`, `entityId` or anything else

Reply:
Yes this name is not good. It made me think why I needed it in the first place:
- if different entities don't use the same approach, then a single `rowId` setting doesn't work. E.g. `customer.customerId` 
 and `order.orderId`.
- a `rowId` setting doesn't work if an entity has a compound key.
- this is really specific from my code base where all entities have an `id` column. I can also just add `id` as a hidden column to my grids.

So I'll remove it.